src/fields/LexicalRichText/index.tsx

```diff
@@ -1,10 +1,11 @@
 import React, { Suspense, lazy } from 'react';
 
-import { Props } from './types';
 import { ShimmerEffect } from 'payload/dist/admin/components/elements/ShimmerEffect';
 
+import { type Props } from './types';
+
 const LexicalRichTextFieldComponent2 = lazy(
-  () => import('./PayloadLexicalRichTextFieldComponent'),
+  async () => await import('./PayloadLexicalRichTextFieldComponent'),
```

Comment:
same as here, why async / await? I think this one is copied straight from the payload core